installer/PowerToysSetup/Product.wxs

```diff
@@ -783,6 +778,7 @@
   <Fragment>
     <ComponentGroup Id="LauncherComponents">
       <Component Id="launcherInstallComponent" Directory="LauncherInstallFolder" Guid="5E688DB4-C522-4268-BA54-ED1CDFFE9DB6">
+        <File Source="$(var.BinX64Dir)modules\Launcher\Microsoft.Launcher.dll" />
```

Comment:
Casing: Launcher -> launcher. The Microsoft.Launcher.dll is being created there and would therefore throw a file not found error during MSI generation.

Reply:
Windows has a case insensitive filesystem though!

Reply:
Windows has a case insensitive filesystem though, so it doesn't cause any problems. Is there a branding reason why we use "launcher"?

Reply:
Thanks for pointing that out. No reason in particular, it was just named that in that in the initial stages of development and has remained the same since.